### DISCUSSION TITLE
Need Help to do Device profiling

### DISCUSSION THREAD

**Discussion by kumardeepak5** [edgexfoundry/community#194] [Q&A]
Hi Team, 

I am trying to Collect AQI Device data into Edgex .

I am expecting AQI Monitoring sensor will produce data in below format
{
    "location": "City Center",
    "timestamp": "2023-10-25T12:30:00",
    "aqi": 76,
    "pm2.5": 25.4,
    "pm10": 38.7,
    "o3": 0.031,
    "co": 0.8,
    "no2": 0.014,
    "so2": 0.006
}

SO for the profiling of same device do i need to create single resource of type Object or i need to create multiple resources  as per properties defined into above json.

Actualy i want to use Devie rest service to collect data with autoevent.

please help me for same.

**Comment by cloudxxx8** [comment]
We added the new feature of using auto event with object data type in REST Device Service from the previous release.  You may check the documentation and try it
https://github.com/edgexfoundry/device-rest-go/blob/main/README.md
